### PR TITLE
Update build scripts to support building iphone components

### PR DESCRIPTION
**Pull request by mrvoorhe**
* iphone runtime

* iphone cross compiler (OSX only)

* iphone simulator

* Add the other build scripts that katana builds run for other runtimes.  These scripts do nothing for now.  The goal is to get a green ACompleteBuild.

### CODE REVIEW

File: add_to_build_results/monodistribution/bin/cli_x86 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. $(dirname $0)/mono-env
```
Comment (joncham) [inline add_to_build_results/monodistribution/bin/cli_x86:2]: Do we want the equivalent *.bat file for windows? Or alternate shell script for Linux?

File: build_runtime_stv.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# The build configs are looking for this file rather than the perl script.
+# easier to add this 
+
+# Note : Not Implemented yet.  Script is here to make the katana build pass so that the mono build artifact is created
+mkdir builds
```
Comment (joncham) [inline build_runtime_stv.sh]: May want newlines at end of files.

**Review by joncham — APPROVED** [review]
LGTM

File: add_to_build_results/monodistribution/bin/cli_x86 (hunk 1 above)
Comment (mrvoorhe) [reply add_to_build_results/monodistribution/bin/cli_x86:2]: Spoke with @joncham , we are going to let this go for now.  I was just adding this to help @lucasmeijer

File: build_runtime_stv.sh (hunk 2 above)
Comment (mrvoorhe) [reply build_runtime_stv.sh]: Good idea

**Pull request merged**